package.json

```diff
@@ -23,7 +23,7 @@
   "scripts": {
     "clean": "rm -rf cache out",
     "lint": "pnpm lint:sol && pnpm prettier:check",
-    "verify": "certoraRun certora/certora.conf",
+    "verify": "certoraRun certora/CommunityTokenDeployer.conf",
```

Comment:
This needs to be changed.
Right now this drops the run of the original CollectibleV1 specs.

We should rather extend this so that `pnpm verify` runs any specs provided in the `certora` folder.

Reply:
Addressed this with https://github.com/status-im/communities-contracts/pull/26/commits/77041256b1380437fa57d7f5fcfa9605af30ab1a which now introduces `pnpm verify:sub_commands` which can be run individually, while still being able to run `pnpm verify` for all specs on CI.